ui/src/hooks/useShipnoiseApi.ts

```
@@ -0,0 +1,179 @@
+
+export async function fetchClipsSearch(
+  params: ClipsSearchParams,
+  signal?: AbortSignal,
+): Promise<ClipsSearchResponse> {
+  const searchParams = new URLSearchParams({
+    shipname: params.shipname,
+    start_date: params.startDate,
+    end_date: params.endDate,
+    limit_per_site: String(params.limitPerSite ?? 5),
+  });
+  (params.sites ?? SITE_VALUES).forEach((site) =>
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)